pkgs/os-specific/linux/system76-io/default.nix

```diff
@@ -0,0 +1,37 @@
+{ stdenv, fetchFromGitHub, kernel }:
+let
+  version = "1.0.1";
+  sha256 = "0qkgkkjy1isv6ws6hrcal75dxjz98rpnvqbm7agdcc6yv0c17wwh";
+in
+stdenv.mkDerivation {
+  name = "system76-io-module-${version}-${kernel.version}";
+
+  passthru.moduleName = "system76_io";
```

Comment:
I have these lines so that nixos-hardware can refer to them rather than having magic strings.  This system76-io module doesn't automatically load itself, so we need to add it to `boot.kernelModules`.

Reply:
ok.